packages/cart-sdk/hooks/use-cart.ts

```diff
@@ -37,8 +37,8 @@ function isValidCartPayload(data: any): data is CartAPIResponse {
 function createCart(input: APICart): Cart {
    const lineItems = input.products.map<CartLineItem>((product) => {
       const priceAmount = parseFloat(product.subPrice);
-      const singleItemDiscount = product.discount
-         ? parseFloat(product.discount)
+      const singleItemDiscount = product.retailDiscount
+         ? parseFloat(product.retailDiscount)
```

Comment:
This already falls back to the user discount, correct? So we don't have to do any fallbacks here?

Reply:
Correct, see

https://github.com/iFixit/ifixit/blob/bd505ceda2824dd50f5c6b5b0de0d329b0918a05/Libs/Cart/BasePricingLib.php#L66-L68

Reply:
I have confirmed that this is the case.